conf/airframes/tudelft/ardrone2_opticflow_indi.xml

```diff
@@ -37,6 +37,9 @@
     <module name="pose_history"/>
     <module name="cv_opticflow">
       <define name="OPTICFLOW_CAMERA" value="bottom_camera"/>
+      <!-- ARDrone2 FPS improvements -->
+	  <define name="OPTICFLOW_PYRAMID_LEVEL" value="0"/>
```

Comment:
Remove extra tab here in all updated files.

Reply:
Strange, this is what I see in eclipse, but on github there is an extra tab. I will run the fix code style to try and fix it!
![screenshot from 2018-01-16 12-02-59](https://user-images.githubusercontent.com/2298791/34985704-459e2cba-fab5-11e7-9a78-637421c0af4d.png)

Reply:
Nevermind, fix code style is not for xml *facepalm*